docker/demo_server/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       SMTP_PASSWORD: '${SMTP_SECRET}'
       WEB_HOST: '${WEB_URL}'
       JWT_SECRET: '${JWT_SECRET}'
+      BACKUPS_PASSWORD: '${BACKUPS_PASSWORD}'
```

Comment:
Oops -- forgot to pass this through, so that's why the backups weren't being encrypted for Fiji.

Have made this fix manually on the server already, so this just persists in the repo for future. (Will need to update other servers too)

Reply:
Ok - I'll just leave this on a note in our group chat so we remember about this next time we update the other server